fails to configure config.h.in when paths contain spaces

When the path contains spaces `configure_file()` is unable to find the template:

```
CMake Error: File /home/dthomas/ros/hydro/desk\ top/sr\ c/geometric_shapes/test/resources/config.h.in does not exist.
CMake Error at test/CMakeLists.txt:6 (configure_file):
  configure_file Problem configuring file
```

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -3,7 +3,7 @@ if(WIN32)
     # Correct directory separator for Windows
     string(REPLACE "\\" "\\\\" TEST_RESOURCES_DIR "${TEST_RESOURCES_DIR}")
 endif(WIN32)
-configure_file("${TEST_RESOURCES_DIR}/config.h.in" "${CMAKE_CURRENT_BINARY_DIR}/resources/config.h")
+configure_file(resources/config.h.in "${CMAKE_CURRENT_BINARY_DIR}/resources/config.h")
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 catkin_add_gtest(test_point_inclusion test_point_inclusion.cpp)

